packages/account/src/providers/chains.ts

```diff
@@ -6,5 +6,6 @@ export const CHAIN_IDS = {
   fuel: {
     devnet: 0,
     testnet: 0,
+    mainnet: 9889,
```

Comment:
Where did this come from?

Reply:
https://github.com/FuelLabs/fuels-ts/issues/2509#issuecomment-2329371444

Reply:
https://github.com/FuelLabs/chain-configuration/tree/master/ignition#the-configuration-of-the-ignition-network